stable/index.yaml

```diff
@@ -1526,7 +1548,7 @@ entries:
   nms-hybrid:
   - apiVersion: v2
     appVersion: 2.10.0
```

Comment:
Shouldn't there be an `nms-hybrid` version update too?

Reply:
ah, yes, let me fix. I've copied the file after the helm command, my bad. thanks!

Reply:
should be fixed